staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js

```diff
@@ -0,0 +1,210 @@
+    $stepUsername.classList.remove('hidden') // hacemos que se muestre el elemento eliminando la clase hidden
+})
+
+
+// Ingresar nombre y hacer click en "EMPEZAR"
+$btnUserName.addEventListener("click", function(){
+   
+    userName = $userName.value;
+    $stepUsername.classList.add('hidden') // ocultamos esta seccion añadiendo la clase hidden al elemento
+
+    // aquí activo el juego
+    $stepGame.classList.remove('hidden') // mostramos el juego quitándole la clase hidden
+
+    // restaurar visibilidad cuando le den a play again y vuelva a salir
+    $btnAnswer.style.display = ""
+    $btnPasapalabra.style.display = ""
+    $userAnswer.style.display = ""
+    $btnPlayAgain.style.display = "none"
+
+    updateLetters()
+    $questions.innerHTML = getNextQuestion()
+
+})
+
+
+// crear letras
+
+function updateLetters(){
+    $letters.innerHTML = ""
+    pasapalabrasGame.forEach(function(letterInfo){
+        let className = "letter"
+        console.log(letterInfo.status)
+        if (letterInfo.status === 1){
+            className += " rightAnswer"  
+        } else if(letterInfo.status === -1){
+            className += " wrongAnswer"
+        }
+        $letters.innerHTML += "<div class='" + className + "'>" + letterInfo.letter + "</div>"         
+    })   
+}
+
+
+// Get next Question cuando hacen click en contestar o pasapalabra
+
+function getNextQuestion(){
+    let listOfQuestion = pasapalabrasGame.slice(actualIndex)
+    console.log(listOfQuestion)
+    let nextQuestion = listOfQuestion.find(letterInfo => letterInfo.status === 0)
+    
+    if (nextQuestion){
+            actualIndex = pasapalabrasGame.findIndex(letterInfo => letterInfo.letter === nextQuestion.letter)
+            console.log(actualIndex)
+           return nextQuestion.question
+    } else {
+        actualIndex = pasapalabrasGame.findIndex(letterInfo => letterInfo.status === 0)
+        nextQuestion = pasapalabrasGame[actualIndex].question
+            return nextQuestion
+    }
+}
+
+
+// check Answers 
+$btnAnswer.addEventListener("click", function(){
+    console.log(pasapalabrasGame[actualIndex].answer)
+
+    if (pasapalabrasGame[actualIndex].answer === $userAnswer.value){
+        pasapalabrasGame[actualIndex].status = 1
+        numOfRightAnswer++
+
+    } else {
+        pasapalabrasGame[actualIndex].status = -1
+        numOfWrongAnswer++
+    }
+
+    if (pasapalabrasGame.length === (numOfRightAnswer + numOfWrongAnswer)){
+        updateLetters()
+        $btnAnswer.style.display = "none"
+        $btnPasapalabra.style.display = "none"
+        $userAnswer.style.display = "none"
+        
+        //sort del ranking
+        playersResults.push({player: userName, score: numOfRightAnswer}) 
+        rankingPlayers = sortRanking(playersResults)
+
+        $questions.innerHTML =  userName + ", has llegado al final del juego. <br><br>" + "<strong>Ranking de jugadores:</strong><br>" + rankingPlayers + "<br>"
```

Comment:
@codingwithdani double ```<br>```'s kill a lot of kitties :( 
You can make a title with a  ```<p>```  and then place the ranking into an  ```<ul>```  with your desired markup

Reply:
@edfrias, done! Creo que se salvaron los gatos 🐱 thanks!